tools/backups/db-backup/mongodb-backup.dc.yaml

```diff
@@ -0,0 +1,185 @@
+          imageChangeParams:
+            automatic: true
+            containerNames:
+              - ${NAME}
+            from:
+              kind: ImageStreamTag
+              namespace: ${IMAGE_NAMESPACE}
+              name: ${NAME}:${IMAGE_TAG}
+      replicas: 1
+      revisionHistoryLimit: 5
+      test: false
+      selector:
+        name: ${NAME}
+      template:
+        metadata:
+          name: ${NAME}
+          labels:
+            name: ${NAME}
+            app: nrpti
+            role: backup
+        spec:
+          volumes:
+            - name: nrpti-${NAME}
+              persistentVolumeClaim:
+                claimName: nrpti-${NAME}
+            - name: nrpti-${NAME}-verification
+              persistentVolumeClaim:
+                claimName: nrpti-${NAME}-verification
+            - name: ${NAME}-config-volume
+              configMap:
+                name: ${NAME}-conf
+                items:
+                  - key: backup.conf
+                    path: backup.conf
+                defaultMode: 420
+          containers:
+            - name: ${NAME}
+              image: image-registry.openshift-image-registry.svc:5000/${IMAGE_NAMESPACE}/${NAME}:${IMAGE_TAG}
+              env:
+                - name: BACKUP_STRATEGY
+                  value: rolling
+                - name: BACKUP_DIR
+                  value: /backups/
+                - name: DAILY_BACKUPS
+                  value: "7"
+                - name: WEEKLY_BACKUPS
+                  value: "5"
+                - name: MONTHLY_BACKUPS
+                  value: "12"
+                - name: MONGODB_AUTHENTICATION_DATABASE
+                  valueFrom:
+                    secretKeyRef:
+                      name: nrpti-mongodb
+                      key: MONGODB_DATABASE
+                - name: DATABASE_USER
+                  valueFrom:
+                    secretKeyRef:
+                      name: nrpti-mongodb
+                      key: MONGODB_USERNAME
+                - name: DATABASE_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: nrpti-mongodb
+                      key: MONGODB_PASSWORD
+                - name: ENVIRONMENT_FRIENDLY_NAME
+                  value: ${ENVIRONMENT_FRIENDLY_NAME}
+                - name: ENVIRONMENT_NAME
+                  value: ${ENVIRONMENT_NAME}
+                - name: WEBHOOK_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: webhook-url
+                      name: backup-webhook
+              resources:
+                limits:
+                  cpu: 250m
+                  memory: 500Mi
+                requests:
+                  cpu: 10m
+                  memory: 50Mi
+              volumeMounts:
+                - name: nrpti-${NAME}
+                  mountPath: /backups/
+                - name: nrpti-${NAME}-verification
+                  mountPath: /var/lib/pgsql/data
```

Comment:
Can't recall if the backup scripts care about the specific paths for the db's. Current DC has this set as `/var/lib/mongodb/data`

Reply:
Yes you're correct!  It should be `/var/lib/mongodb/data`.  Fixed in https://github.com/bcgov/NRPTI/pull/757/commits/09cee3a4cf7c244a41ce53edeb00a800b02594a6